g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/utils/S3Util.java

```diff
@@ -123,31 +121,36 @@ public void unlockFile() {
 			}
 		}
 
-		private boolean waitLockfileDisappeared(long endTime) {
+		private boolean waitLockfileDisappeared() {
 			boolean bDeadlockTested = false;
-			List<S3ObjectSummary> objects = s3Client.getS3Client().listObjectsV2(config.getBucketName(), this.key).getObjectSummaries();
-			while (!objects.isEmpty()) {
+			boolean existed = s3Client.getS3Client().doesObjectExist(config.getBucketName(), this.key);
+			int i = 0;
```

Comment:
I think time is more intuitionistic than number. We don't know how long it will wait by a number.